### ISSUE TITLE
[BUG] - .shape() method not accounting for different grid sizes

### ISSUE THREAD

**Issue by otigon** [fantasycalendar/FoundryVTT-Sequencer#140]
**Describe the bug**
When using the ``.shape()`` method under ``.effect()``, setting ``gridUnits: true`` does not account for different grid sizes.

**To Reproduce**
1. Use the example macro located on the Wiki page at https://fantasycomputer.works/FoundryVTT-Sequencer/#/api/effect?id=shape
2. Test first on a 100 pixel grid size map. Works as expected
3. Change grid size on the map to 200 pixels.
4. The radius of the circle is no longer matched to the grid units

**Expected behavior**
``gridUnits: true`` would account for different grid sizes?

**Screenshots**
N/A

**Setup:**
 - Foundry Version: 10.291
 - System Version: DND5e 2.1.4
 - Sequencer Version: 2.411
 - Google Chrome 109.0.5414.120

**Active modules:**
 - Sequencer
 - socketlib
 - Automated Animations

**Additional context**
Macro from Wiki:
```js
new Sequence()
    .effect()
        .attachTo(token)
        .persist()
        .shape("circle", {
            lineSize: 4,
            lineColor: "#FF0000",
            radius: 1.5,
            gridUnits: true,
            name: "test"
        })
        .loopProperty("shapes.test", "scale.x", { from: 0.9, to: 1.1, duration: 1000, pingPong: true, ease: "easeInOutSine" })
        .loopProperty("shapes.test", "scale.y", { from: 0.9, to: 1.1, duration: 1000, pingPong: true, ease: "easeInOutSine" })
    .play()
```

**Comment by otigon** [comment]
I *think* this is happening because of line 393 of ``canvas-effect.js``. You have ``gridSize: 100``. Changing this to ``gridSize: canvas.grid.size`` corrects the sizing issue. However, I've no idea if that borks stuff elsewhere

**Comment by otigon** [comment]
I was playing around with this more today, and decided to try a work-around and not use anything with ``gridUnits: true``, so just passing pixel size. However, this still was throwing it off.

In ``canvas-lib.js`` lines 15, 16 and 17 I removed the ``gridSizeDifference`` divisor and it started functioning correctly when not passing ``gridUnits: true``.

```js
  const offsetX = (shape.offset?.x ?? 0) * (shape.offset?.gridUnits ? canvas.grid.size : 1)// / gridSizeDifference;
  const offsetY = (shape.offset?.y ?? 0) * (shape.offset?.gridUnits ? canvas.grid.size : 1)// / gridSizeDifference;
  const sizeMultiplier = (shape.gridUnits ? canvas.grid.size : 1)// / gridSizeDifference;
```

I reverted all my A-A changes and went back to the normal method of using ``gridUnits: true`` (with the ``gridSizeDifference`` commented out as above), and this also is working as intended.

So I *think* that is the culprit, but again I'm uncertain what kind of cascading effects that will have. Hope this helps to narrow it down
